src/components/benchmarks/Benchmarks.tsx

```diff
@@ -0,0 +1,183 @@
+          fill="#20159E"
+        />
+        <path
+          d="M12.4971 18.498C12.2087 18.4986 11.928 18.4053 11.6973 18.2324L11.683 18.2216L8.6708 15.9154C8.5313 15.8085 8.41423 15.6752 8.32625 15.5231C8.23828 15.3709 8.18112 15.203 8.15806 15.0287C8.13499 14.8545 8.14648 14.6775 8.19184 14.5077C8.23721 14.3379 8.31557 14.1787 8.42246 14.0392C8.52934 13.8997 8.66265 13.7826 8.81478 13.6947C8.96692 13.6067 9.13489 13.5495 9.30911 13.5265C9.48333 13.5034 9.66039 13.5149 9.83017 13.5603C9.99996 13.6056 10.1591 13.684 10.2986 13.7909L12.2497 15.2869L16.8603 9.27395C16.9673 9.13448 17.1006 9.01745 17.2528 8.92952C17.405 8.8416 17.5729 8.78451 17.7472 8.76151C17.9214 8.73851 18.0984 8.75005 18.2682 8.79548C18.438 8.84091 18.5971 8.91933 18.7366 9.02627L18.7079 9.0652L18.7373 9.02684C19.0187 9.24304 19.2027 9.56202 19.2492 9.91377C19.2956 10.2655 19.2006 10.6213 18.985 10.9031L13.5618 17.9758C13.4364 18.1388 13.2751 18.2707 13.0904 18.3612C12.9058 18.4518 12.7028 18.4986 12.4971 18.498Z"
+          fill="white"
+        />
+      </svg>
+      <h1 style={{ fontWeight: "bold", fontSize: "18px" }}>
+        {blurb || "No benchmarks yet"}
+      </h1>
+      <span style={{ marginBottom: "6px" }}>
+        Create student benchmarks to start tracking their goals
+      </span>
+      <Box>
+        <Link className={$button.default} href={`${router.asPath}/create`}>
+          Add benchmark
+        </Link>
+      </Box>
+    </Grid>
+  );
+};
+
+export default function Benchmarks({ subgoals }: { subgoals: Subgoal[] }) {
+  const router = useRouter();
+  const [activeTab, setActiveTab] = useState<selectionValue>(
+    selectionValue.ALL
+  );
+  return (
+    <Stack sx={{ width: 1 }}>
+      {/* tabs */}
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: { xs: "column-reverse", sm: "row" },
+          justifyContent: "space-between",
+        }}
+      >
+        <Box className={$GoalPage.benchmarksBox} justifyContent="flex-start">
+          <SelectableTab
+            text={`All (${subgoals.length})`}
+            value={selectionValue.ALL}
+            handleSelect={() => {
+              setActiveTab(selectionValue.ALL);
+            }}
+            currentlySelected={activeTab}
+          />
+
+          <SelectableTab
+            text={`Completed (${
+              subgoals.filter((goal) => goal.status === "Complete").length
+            })`}
+            value={selectionValue.COMPLETE}
+            handleSelect={() => {
+              setActiveTab(selectionValue.COMPLETE);
+            }}
+            currentlySelected={activeTab}
+          />
+        </Box>
+        <br />
+        <Box>
+          <Link className={$button.default} href={`${router.asPath}/create`}>
+            Add benchmark
+          </Link>
+        </Box>
+      </Box>
+      {/* TODO: Populate Benchmarks container */}
```

Comment:
Seems like this comment should be removed

Reply:
Removed comment